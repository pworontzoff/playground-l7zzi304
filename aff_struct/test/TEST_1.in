Dupond
Jean
rue judong
5
Liege
4000
6
6
1945
