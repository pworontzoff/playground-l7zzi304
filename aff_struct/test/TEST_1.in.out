Dupond Jean
rue judong 5
4000 Liege
Date de naissance: 06/06/1945